UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -69,6 +91,9 @@ protected void onCreate(Bundle savedInstanceState) {
         getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
         setContentView(R.layout.activity_driver_dashboard);
 
+        bottomAwayAnim = AnimationUtils.loadAnimation(this, R.anim.slide_away_down_animation);
```

Comment:
I can't seem to simulate the animation on my end. Is this working as intended?

https://user-images.githubusercontent.com/38304351/119358267-bf016780-bca8-11eb-9b2f-7c57f13d8041.mov

Reply:
Works like this locally (with the changes you required, going to push in a minute)
https://user-images.githubusercontent.com/71485862/119360514-fb35c780-bcaa-11eb-8d00-532199942d7c.mov

Reply:
And yeah it is, just did not rename the animation. I thought about sliding away but then I thought it get in the way of the thumbstick. I can rename the animation for more clarity.

Reply:
Ah I see, the animation/ list didn't pop up for me even when I pushed the previous "NEXT" button.  Will test again, thanks.

Oh don't change the naming! I was referring to its functionality, since I can see it instantiated in the code but it didn't show anything in the simulator.